index.tsx

```diff
@@ -79,14 +80,16 @@ function MasonryList<T>(props: Props<T>): ReactElement {
       contentContainerStyle={contentContainerStyle}
       removeClippedSubviews={removeClippedSubviews}
       refreshControl={
-        <RefreshControl
-          refreshing={!!(refreshing || isRefreshing)}
-          onRefresh={() => {
-            setIsRefreshing(true);
-            onRefresh?.();
-            setIsRefreshing(false);
-          }}
-        />
+        props.refreshControl === false ? null : (
```

Comment:
Why not just `!props.refreshControl` instead of `props.refreshControl === false`?

Reply:
@Conradmaker Could you kindly answer this? 🙏